chat/public/js/src/chat.js

```diff
@@ -24,6 +24,22 @@ function ChatTemplates(init_data) {
         );
     };
 
+    var noticeTemplate = function(step) {
+      var tag = 'div.notice';
+
+      if (step.notice_type)
```

Comment:
We currently don't have a formal JS style guide, but please always use curly braces in if/else blocks.